.circleci/config.yml

```diff
@@ -46,4 +46,3 @@ workflows:
     jobs:
       - "python-3.6"
       - "python-3.7"
-      - "upstream-dev"
```

Comment:
I don't want to debug this problem for now: https://circleci.com/gh/krivonogov/fletcher/2?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Reply:
https://issues.apache.org/jira/browse/ARROW-6844